### PR TITLE
rewrite spring-boot binding

### PR DESCRIPTION
**Pull request by tokuhirom**
- breaks backward compatibility
- split modules
-

### CODE REVIEW

File: spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    //            final CompletableFuture<List<TextMessage>> returnValue = new CompletableFuture<>();
+    //            target.accept(returnValue);
+    //            returnValue.complete(singletonList(new TextMessage("Reply Text")));
+    //        });
+    //
+    //        // Verify
+    //        assertThat(systemOut)
+    //                .contains("failed")
+    //                .contains("EXCEPTION HAPPEN!");
+    //    }
```
Comment (Yang-33) [inline spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java:153]: Is this still necessary...?

File: spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        final String channelToken = lineBotProperties.channelToken();
+        return FixedChannelTokenSupplier.of(channelToken);
+    }
+
+    /**
+     * Expose {@link MessagingApiClient} as {@link Bean}.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public MessagingApiClient messagingApiClient(
```
Comment (Yang-33) [inline spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java:61]: I forgot `manager.line.biz` client in https://github.com/line/line-bot-sdk-java/pull/956. Do you have any plan to support it?

Comment (tokuhirom) [reply spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java:61]: Oops. I forgot it.  I'll implement it in next PR.

File: spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java (hunk 1 above)
Comment (tokuhirom) [reply spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java:153]: i'll check it

File: spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java (hunk 2 above)
Comment (Yang-33) [reply spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java:61]: Thank you!

File: spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java (hunk 1 above)
Comment (tokuhirom) [reply spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java]: I removed this. This test case is meaningless...

**Pull request merged**